### ISSUE TITLE
Create forwards / disambiguation-page for commands in platform sub-dirs

### ISSUE THREAD

**Issue by FreifunkerEZ** [tldr-pages/tldr#1403]
Hellos.
TLDR: Searched for "rpm" found nothing. Should have searched for "linux/rpm". Fix that.

I was trying to find out about 'rpm' and was kinda surprised that it did not exist.
https://tldr.ostera.io/rpm

But hey, an opportunity to give back! Reading through all the contribution docs, forking, cloning and branching. And I look at the files and find sub-directories. And in them I find 'rpm'. And so I was able to find
https://tldr.ostera.io/linux/rpm

And I wish that there was a page at common/rpm that forwarded me to linux/rpm. And if there were pages on multiple platforms, that it would give me links to all of them. Like in wikipedia.

Or maybe it's just a client issue with not searching multiple namespaces but just looking in a single location directly determined by the  user's input? Dunno. You decide. Knowing about platform-dirs and when to use them makes tldr-pages super complicated for me. (super meaning: from "totally not complicated" to "a little bit complicated" which is an increase of a million-billion percent. Thus superTM)

Thanks for the pages and cheers!
chris :)

**Comment by agnivade** [comment]
> Or maybe it's just a client issue with not searching multiple namespace

You are right ! Indeed this seems to be an issue with the web client. The node client automatically searches for platform specific pages first, and then looks under the common folder. So, your concern is already handled in the node client.

I would recommend using the node client, or raising an issue at https://github.com/ostera/tldr.jsx to get a discussion going.

**Comment by FreifunkerEZ** [comment]
Interesting! Will do. Thanks :)
